docs/template.html

```diff
@@ -52,6 +52,7 @@
     <link rel="stylesheet" href="/styles.css" />
     <script src="/action.js"></script>
     <script src="/tabs.js"></script>
+    <script async src="https://cse.google.com/cse.js?cx=a49b06597fd4f470d"></script>
```

Comment:
Could you add a comment to explain where you got this from?

Reply:
Sure thing, done.